Mathlib/Order/Grade.lean

```diff
@@ -326,3 +327,60 @@ instance [GradeOrder ℕ α] : WellFoundedLT α :=
 
 instance [GradeOrder ℕᵒᵈ α] : WellFoundedGT α :=
   GradeOrder.wellFoundedGT ℕᵒᵈ
+
+end Preorder
+
+/-!
+### Grading a flag
+
+A flag inherits the grading of its ambient order.
+-/
+
+namespace Flag
+variable [PartialOrder α] {s : Flag α}
+
+@[simp]
+lemma coe_covby_coe {a b : s} : (a : α) ⋖ b ↔ a ⋖ b := by
```

Comment:
```suggestion
lemma coe_covBy_coe {a b : s} : (a : α) ⋖ b ↔ a ⋖ b := by
```
is there a wcovBy version?

Reply:
Oh I wrote this code before `wcovBy` was a thing. Will add